packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -3,20 +3,38 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-alert-background-color: Specifies the background color of the component.
+ * @prop --calcite-alert-border-color: Specifies the border color of the component.
+ * @prop --calcite-alert-chip-background-color: Specifies the background color of the queue count chip in the component.
+ * @prop --calcite-alert-chip-border-color: Specifies the border color of the queue count chip in the component.
+ * @prop --calcite-alert-chip-corner-radius: Specifies the corner radius of the queue count chip in the component.
+ * @prop --calcite-alert-chip-text-color: Specifies the color of the queue count chip text in the component.
+ * @prop --calcite-alert-close-background-color-active: Specifies the background color of the close button in the component when the close button is active.
+ * @prop --calcite-alert-close-background-color-focus: Specifies the background color of the close button in the component when the close button is focused.
+ * @prop --calcite-alert-close-background-color-hover: Specifies the background color of the close button in the component when the close button is hovered.
+ * @prop --calcite-alert-close-background-color: Specifies the background color of the close button in the component.
+ * @prop --calcite-alert-close-icon-color-active: Specifies the color of the close button icon in the component when the close button is active.
+ * @prop --calcite-alert-close-icon-color-focus: Specifies the color of the close button icon in the component when the close button is focused.
+ * @prop --calcite-alert-close-icon-color-hover: Specifies the color of the close button icon in the component when the close button is hovered.
+ * @prop --calcite-alert-close-icon-color: Specifies the color of the close button icon in the component.
+ * @prop --calcite-alert-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-alert-link-text-color: Specifies the color of the link text in the component.
```

Comment:
Because these are all slotted (link, message, title) - can we just set defaults and have users override on their slotted content (no css prop?)

Reply:
That's **way** cleaner. ✨🧹✨
![](https://i.imgur.com/7IeuRzt.gif)

We could turn your above suggestion into a guideline.

For cases where a component needs to apply specific overrides to slotted content:

* set variables/overrides on the slotted elements
* ☝️ would not necessitate component-level CSS props
* users can override the slotted elements themselves if needed

cc @alisonailea

Reply:
I *do* think we should have some more consistency across these implementations. I'd like to see props for heading / description as well as slots - often times even simple text is slotted in and contains extra padding / margins / etc., that aren't desirable. Users can still of course have the option of slotted content when needed for rich content like links, images, etc.

But in places like Alert, Notice, Card, etc., it could be nice to have the properties as a first line of defense for consistency, with slots there as option for more advanced use cases. Of course that's out of scope for this PR / effort.

Reply:
(and, now that I type it out - continuing to re-name all these text lockups to heading / description where applicable - right now we have a mix of that, and title / subtitle / label / etc.)